rsts/user/getting_started/create_first.rst

```diff
@@ -4,122 +4,101 @@
 Writing Your First Workflow
 ########################################
 
+By the end of this getting started guide you'll be familiar with how easy it is to author a Flyte workflow, run and then deploy it locally.
+
 The easiest way to author a Flyte Workflow is using the provided python SDK called "FlyteKit".
 
-You can save some effort by cloning the ``flytesnacks`` repo, and re-initializing it as a new git repository ::
+You can save some effort by cloning the ``flytekit-python-template`` repo, and re-initializing it as a new git repository ::
 
-  git clone git@github.com:lyft/flytesnacks.git myflyteproject
+  git clone git@github.com:lyft/flytekit-python-template.git myflyteproject
   cd myflyteproject
   rm -rf .git
   git init
   cd python
```

Comment:
```suggestion
  cd myflyteproject
```

Reply:
i'll just remove. we should already be in myflyteproject